Basic-Car-Maintenance/Shared/Odometer/Views/OdometerView.swift

```diff
@@ -99,6 +101,29 @@ struct OdometerView: View {
             }
             .toolbar {
                 ToolbarItemGroup(placement: .primaryAction) {
+                    Menu {
+                        Button("All") {
+                            selectedVehicleID = "nil"
+                            applyFilter()
+                            print(selectedVehicleID!)
+                        }
+                        ForEach(viewModel.vehicles) { value in
```

Comment:
Instead of using the variable name `value`, use `vehicle` because it's more descriptive